stable/xray/templates/xray-server-statefulset.yaml

```diff
@@ -147,6 +147,11 @@ spec:
         volumeMounts:
         - name: data-volume
           mountPath: "{{ .Values.common.xrayConfigPath }}"
+        {{- if .Values.customCA.enabled }}
+        - name: custom-ca
+          mountPath: "/etc/ssl/certs/custom-ca.crt"
+          subPath: "custom-ca.crt"
```

Comment:
This means that the key has to be set to custom-ca.crt
If that’s how you intend it to be, it should be well documented

Reply:
This is as intended. If you check the README, I call it out specifically.